packages/nimble-components/CONTRIBUTING.md

```diff
@@ -77,15 +77,34 @@ Before building a new component, 3 specification documents need to be created:
 
 ## Develop new components
 
-### Marking a component as in development
+### Marking a component as experimental / in-development / incubating
 
-If a component will require multiple pull requests before having a complete and stable API, it should be marked as "in-development" to indicate to clients that they shouldn't start using it yet. To do this:
+If a component is not ready for general use, it should be marked as "experimental" to indicate that status to clients. A component could be in this state if it:
+
+-   is still in development
+-   is currently application-specific and hasn't yet been generalized for broader use
+-   is missing important features like interaction design, visual design, accessibility, or framework integration
+
+Experimental contributions may compromise on the above capabilities but they still must abide by other repository requirements like:
+
+-   coding conventions (element naming, linting, code quality)
+-   unit and Chromatic test coverage
+-   Storybook documentation
+-   semantic versioning
```

Comment:
Do we want these experimental components to require semantic versioning? Based on what we're doing with the table, I'd think the answer is "no" -- they should be allowed to have breaking changes as much as they want without it being a "major" change until the component is no longer experimental.

Reply:
I was thinking about changing this requirement but Milan convinced me to keep our policy as-is. I removed this line and made sure that policy is documented in a couple other places.